src/main/java/com/marklogic/spark/reader/MarkLogicScanBuilder.java

```diff
@@ -167,12 +166,10 @@ public boolean supportCompletePushDown(Aggregation aggregation) {
             return false;
         }
         AggregateFunc[] expressions = aggregation.aggregateExpressions();
-        if (expressions.length == 1 && expressions[0] instanceof CountStar) {
-            // If a count() is used, it's supported if there's no groupBy - i.e. just doing a count() by itself -
-            // and supported with a single groupBy - e.g. groupBy("column").count().
-            return aggregation.groupByExpressions().length < 2;
-        }
-        return false;
+        // If a count() is used, it's supported if there's no groupBy - i.e. just doing a count() by itself -
```

Comment:
I think I made this needlessly complicated before - since `op.groupBy` supports multiple column names, we don't have to care how many column names the user passed into the Spark `groupBy` call.